data/presets/historic/wayside_cross.json

```
@@ -18,5 +18,8 @@
     "tags": {
         "historic": "wayside_cross"
     },
+    "terms": [
+        "cross"
```

```suggestion
        "cross", 
        "Christian", 
        "Jesus", 
        "Christ"
```

These should alow be added as indicated in the issue? As at least in Germany it's usually a Jesus statue being at the cross.